api/src/formatters/yaml-nested.ts

```diff
@@ -49,6 +49,11 @@ export const yamlNestedExporter: Exporter = async (data: IntermediateTranslation
         if (!current.hasOwnProperty(key)) {
           current[key] = {};
         }
+        if (typeof current[key] === 'string') {
+          throw new Error(
+            `You have a flat key that have both a value and sub keys. This is not allowed on nested JSON. Sub key: ${translation.term}`,
```

Comment:
Could you update the error message to say "This is not allowed on nested YAML"?

Reply:
Fixed it, good catch :)